app/src/main/java/com/example/fitjournal/core/Workouts.kt

```diff
@@ -0,0 +1,47 @@
+package com.example.fitjournal.core
+
+import com.example.fitjournal.core.presentation.model.enums.WorkoutTypeEnum
+import com.example.fitjournal.journalEntry.domain.WorkoutDetail
+
+object Workouts {
```

Comment:
@Alex-A-Fit  I was wondering if the data used for the library is similar to this and it should be merged?

Reply:
either merged or removed as we have a mock data set implemented now